src/common.ts

```diff
@@ -24,4 +25,21 @@ export class Common {
       });
     }
   }
+  public static async getLoginUrl(instanceUrl: Optional<string>): Promise<Optional<string>> {
```

Comment:
nit: I'm not a fan of async methods that start with `get`. I would say either remove async or call it `resolveLoginUrl`